composer.json

```diff
@@ -13,12 +13,13 @@
         }
     ],
     "require": {
-        "php": ">=5.3.9",
+        "php": ">=5.5.0",
```

Comment:
my point is still valid : following and supporting * stable * versions of php is what will help deprecated / unmaintained versions to stay what they should be, dead.

As you are following semver (and even without that actually, as in the semver, it doesn't say you should bump major versions on dependency upgrades...), if someone wants to have a psysh that supports an old version of php, they should just use a previous version that supports it. Then it's up to you to keep or not a support (if it was up to me, well... there should not be any, except one or two previous versions at most)

Especially as psysh is a dev tool. But once again, it's only my opinion, I'll still be using psysh, even though I may not agree with some points, as it is still awesome :}